astropy/cosmology/_io/tests/test_ecsv.py

```diff
@@ -3,9 +3,10 @@
 # THIRD PARTY
 import pytest
 
+from astropy.cosmology._io.ecsv import read_ecsv, write_ecsv
+
 # LOCAL
```

Comment:
The comments here are lying - renaming a local module did not turn it into a third party module.

Reply:
Agreed. Removing all instances of the import location comments in ``cosmology/_io``.